ULWGL/ulwgl_util.py

```diff
@@ -11,7 +11,7 @@
 from sys import stderr
 from pathlib import Path
 from pwd import struct_passwd, getpwuid
-from shutil import rmtree, move, copyfile, which
+from shutil import rmtree, move, copyfile, copy, which
 
```

Comment:
At this point this PR is now going outside it's intended scope and since 4c18061049c70c2691be8e15fd961b88dba38a6e we're no longer creating shallow copies. 

Please revert

Reply:
I will